fe/fe-core/src/main/java/org/apache/doris/mtmv/BaseTableInfo.java

```diff
@@ -171,7 +171,11 @@ public void compatible(CatalogMgr catalogMgr) {
             this.dbName = db.getFullName();
             this.tableName = table.getName();
         } catch (AnalysisException e) {
-            LOG.warn("MTMV compatible failed, ctlId: {}, dbId: {}, tableId: {}", ctlId, dbId, tableId, e);
+            String msg = String.format(
+                    "Failed to get name based on id during compatibility process, ctlId: %s, dbId: %s, tableId: %s",
+                    ctlId, dbId, tableId);
+            LOG.warn(msg, e);
+            throw new Exception(msg);
```

Comment:
maybe we should use return value, not throw exception. throw exception is very expansive